.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
+    "trailingComma": "all",
+    "printWidth": 80,
+    "tabWidth": 2,
+    "useTabs": false
```

Comment:
Does this automatically update the developer's settings for alt-shift-f? Or does each developer have to manually update his config

Reply:
For prettier, you will need the vscode extension for it to detect the settings and format on save. Otherwise, the settings wont be used when formatting by default